buildForDebugger.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# run this script script from the project root.  It will build with g++ -g compiler flags, so
+# gdb can be used to debug
+# Note: This is only tested on Ubuntu 20
+
+mkdir build
```

Comment:
This fails if the directory already exists (which is likely).
```suggestion
mkdir -p build
```

Reply:
Good point, I will add this